### ISSUE TITLE
Unable to run npm run production in laravel 5.4

### ISSUE THREAD

**Issue by sunscreem** [ecrmnn/collect.js#114]
I'm importing collect globally in laravel's default `bootstrap.js` file:

```
window.collect = require('collect.js');
```

When I run `npm run development` everything works fine.

When I run `npm run production` I get the following error:

```
> @ production C:\www\kilt2016
> cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js

 95% emitting ERROR  Failed to compile with 1 errors7:21:22 PM

 error

/js/oa/vue.57617082aa194c799028.js from UglifyJs
Unexpected token: name (index) [./~/collect.js/src/methods/chunk.js:5,0][/js/oa/vue.57617082aa194c799028.js:79,6]

```
(Laravel version: 5.4.24, Mix Version: 0.12.1)

Anyone any ideas?

**Comment by ecrmnn** [comment]
I've just published version 3.0.31.

Could you try to reinstall your node modules? And try again?

```bash
rm -rf node_modules/
npm install
```

**Comment by sunscreem** [comment]
Yup.

That's working a treat now.

Thank you.
